src/drivers/pwm_out_sim/pwm_out_sim.cpp

```diff
@@ -547,6 +575,15 @@ PWMSim::task_main()
 			_armed = aa.armed;
 			_failsafe = aa.force_failsafe;
 			_lockdown = aa.manual_lockdown;
+
+			param_t param_handle;
+			
+			// maximum motor slew rate parameter
+			param_handle = param_find("MOT_SLEW_MAX");
```

Comment:
This results in high-rate param updates. I think setting this on boot is acceptable - you can mark the slew rate param as:

```
@reboot_required true
```